src/Forms/SudoModePasswordField.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\Forms\FormField;
+
+class SudoModePasswordField extends FormField
+{
+    public const FIELD_NAME = 'SudoModePasswordField';
+    
+    protected $schemaComponent = 'SudoModePasswordField';
+
+    protected $extraClasses = [
+        'SudoModePasswordField',
+    ];
+
+    /**
+     * Whether the field should display with padding around it
+     */
+    private bool $panelPadded = true;
```

Comment:
The "panelPadded" functionality in [this PR](https://github.com/silverstripe/silverstripe-cms/pull/3053) is related to making the field looking pretty "inline" which is only used in this PR which I think we should close. If we close that, then the panelPadded functionality should be removed from this PR.